src/main/scala/za/co/absa/springdocopenapiscala/OpenAPIModelRegistration.scala

```diff
@@ -66,6 +67,7 @@ class OpenAPIModelRegistration(components: Components) {
     case t if t <:< typeOf[Option[_]]                                      => handleType(t.typeArgs.head)
     case t if t <:< typeOf[Seq[_]] || t <:< typeOf[Array[_]]               => handleSeqLike(t)
     case t if t <:< typeOf[Set[_]]                                         => handleSet(t)
+    case t if t <:< typeOf[Enumeration#Value]                              => handleEnum(tpe)
```

Comment:
Why do you pass `tpe` here and not `t`?

Reply:
Actually can be `t`. Changed, thanks.

Initially when I was experimenting, I passed `tpe` as it is not dealiased, and I thought it might be needed non-dealiased. But `parentObjectType` is fortunately the same whether dealiased or not, so can be `t`.